modules/ossm-migrating-from-ior-to-explicitly-managed-routes.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+// * service_mesh/v2x/ossm-route-migration.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-migrating-from-ior-to-explicitly-managed-routes_{context}"]
+= Migrating from Istio OpenShift Routing to explicitly-managed routes
+
+This procedure explains how to disable Istio OpenShift Routing (IOR) in {SMProductName}, and how to continue to use and manage Routes that were originally created using IOR. This procedure also provides an example of how to explicitly create a new Route targetting an existing gateway Service.
+
+.Prerequisties
+
+* Before migrating to explicitly-managed routes, export the existing route configurations managed by Istio OpenShift Routing (IOR) to files. Save the files so that in the future you can recreate the route configurations without requiring IOR.
+
+.Procedure
+
+. Modify the `ServiceMeshControlPlane` resource to disable IOR:
```

Comment:
Use unordered list for single-step procedures:
```suggestion
* Modify the `ServiceMeshControlPlane` resource to disable IOR:
```

Reply:
done